e2e/plugin/src/nx-plugin.test.ts

```diff
@@ -75,7 +75,7 @@ describe('Nx Plugin', () => {
       generators: expect.objectContaining({
         [`update-${version}`]: {
           version,
-          description: `update-${version}`,
+          description: `Migration for v1.0.0`,
```

Comment:
Why did this change?

Reply:
It just makes more sense, don't you think?